### PR TITLE
Fix creation time

### PR DESCRIPTION
**Pull request by abhinavkgrd**
## Description
Update creation time with values from EXIF for selected files, (only image files processed)

## Test Plan
- [x] non-image files are skipped
- [x] update skipped if EXIF Time and current creation Time are the same

### CODE REVIEW

File: src/utils/strings/englishConstants.tsx [hunk 1]
```diff
@@ -596,6 +596,23 @@ const englishConstants = {
     REPLACE_THUMBNAIL_COMPLETED_WITH_ERROR: () => (
         <>could not compress some of your thumbnails, please retry</>
     ),
+    FIX_CREATION_TIME: 'fix creation time',
```
Comment (vishnukvmd) [inline src/utils/strings/englishConstants.tsx:599]: `fix time`

File: src/utils/strings/englishConstants.tsx [hunk 2]
```diff
@@ -596,6 +596,23 @@ const englishConstants = {
...
+    CREATION_TIME_UPDATED: `file's creation time updated`,
```
Comment (vishnukvmd) [inline src/utils/strings/englishConstants.tsx:600]: `file time updated`

File: src/utils/strings/englishConstants.tsx [hunk 3]
```diff
@@ -596,6 +596,23 @@ const englishConstants = {
...
+
+    UPDATE_CREATION_TIME_NOT_STARTED: () => (
+        <>update creation time with exif values</>
```
Comment (vishnukvmd) [inline src/utils/strings/englishConstants.tsx:603]: `update file time with values from EXIF`

File: src/utils/strings/englishConstants.tsx [hunk 4]
```diff
@@ -596,6 +596,23 @@ const englishConstants = {
...
+    ),
+    UPDATE_CREATION_TIME_COMPLETED: () => <>successfully updated all files</>,
+    UPDATE_CREATION_TIME_NOOP: () => (
+        <>
+            you have no files that whose CreationTime is not updated with exif
```
Comment (vishnukvmd) [inline src/utils/strings/englishConstants.tsx:608]: `you have no files that whose time is out of sync with exif`

File: src/utils/strings/englishConstants.tsx [hunk 5]
```diff
@@ -596,6 +596,23 @@ const englishConstants = {
...
+            value
+        </>
+    ),
+    UPDATE_CREATION_TIME_COMPLETED_WITH_ERROR: () => (
+        <>updating creation time failed for some files ,try again</>
```
Comment (vishnukvmd) [inline src/utils/strings/englishConstants.tsx:613]: `file time updation failed for some files, please retry`

**Review by ua741 — DISMISSED** [review]

**Review by ua741 — APPROVED** [review]

**Pull request merged**